libqtile/widget/memory.py

```diff
@@ -0,0 +1,32 @@
+import re
```

Comment:
Please add a copyright notice at the top of this file (see the other files for an example)